components/ApplicationTable.tsx

```diff
@@ -46,15 +51,82 @@ const columns = [
     cell: (info) => info.getValue(),
     header: 'Next Action',
     id: 'nextAction'
+  }),
+  columnHelper.accessor('reviewer.loginId', {
+    cell: (info) => info.getValue(),
+    header: 'Reviewer',
+    id: 'reviewer'
   })
 ]
 
 interface ApplicationTableProps {
   applications: ApplicationRow[]
+  reviewerIds: string[]
 }
 
-const ApplicationTable: FC<ApplicationTableProps> = ({ applications }) => (
-  <TanstackTable data={applications} columns={columns} />
-)
+const ApplicationTable: FC<ApplicationTableProps> = ({ applications, reviewerIds }) => {
+  const router = useRouter()
+  const pathname = usePathname()
+  const searchParams = useSearchParams()
+  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([])
+  const [nextActionFilterValue, setNextActionFilterValue] = useState('All')
+  const [reviewerFilterValue, setReviewerFilterValue] = useState('All')
+
+  // searchParams determine what filters should be applied and the value of the dropdown
+  useEffect(() => {
+    setColumnFilters(Array.from(searchParams).map(([key, value]) => ({ id: key, value: value })))
+
+    setNextActionFilterValue(searchParams.get('nextAction') || 'All')
+    setReviewerFilterValue(searchParams.get('reviewer') || 'All')
```

Comment:
You are starting to use `All`, `nextAction` and `reviewer` in a few different places. Constants?

Reply:
constants added in 3993a3331e005e1ae1fa3b79a364ca8ee3c2f24f